app/utils/prepareAnswer.js

```diff
@@ -362,21 +363,46 @@ function summaryGraph(body, n) {
     active.push(
       dailyData[i].dailyconfirmed - dailyData[i].dailydeceased - dailyData[i].dailyrecovered,
     );
-    dateLabel.push(`'${dailyData[i].date}'`);
+    dateLabel.push(`${dailyData[i].date}`);
   }
   confirmed.reverse();
   recovered.reverse();
   deaths.reverse();
   active.reverse();
   dateLabel.reverse();
 
-  let link = `https://quickchart.io/chart?c={type:'line',data:{labels:[${dateLabel}],`;
-  link += `datasets:[{label:'Confirmed',data:[${confirmed}],backgroundColor:'white', fill:false, borderColor:'red'},`;
-  link += `{label:'Active',data:[${active}],backgroundColor:'white', fill:false, borderColor:'blue'},`;
-  link += `{label:'Recovered',data:[${recovered}],backgroundColor:'white', fill:false, borderColor:'green'},`;
-  link += `{label:'Deceased',data:[${deaths}],backgroundColor:'white', fill:false, borderColor:'grey'}]},`;
-  link += `options: {title:{display: true,text: 'Daily cases analysis for past ${minN} days'},}}`;
-  return link;
+  const myChart = new QuickChart();
+  myChart
+    .setConfig({
+      type: 'line',
+      data: {
+        labels: dateLabel,
+        datasets: [{
+          label: 'Confirmed', data: confirmed, backgroundColor: 'white', fill: false, borderColor: 'red',
+        },
+        {
+          label: 'Active', data: active, backgroundColor: 'white', fill: false, borderColor: 'blue',
+        },
+        {
+          label: 'Recovered', data: recovered, backgroundColor: 'white', fill: false, borderColor: 'green',
+        },
+        {
+          label: 'Deceased', data: deaths, backgroundColor: 'white', fill: false, borderColor: 'grey',
+        }],
+      },
+      options: {
+        title: {
+          display: true,
+          text: `Daily cases analysis for past ${minN} days`,
+        },
+      },
+    }).setDevicePixelRatio(2);
+
+  async function gaphWithShortUrl() {
```

Comment:
You don't need this another function just use `await` here.

Reply:
Okay I removed this extra function 👍